cmd/admin-bucket-quota.go

```diff
@@ -118,10 +111,7 @@ func checkAdminBucketQuotaSyntax(ctx *cli.Context) {
 		cli.ShowCommandHelpAndExit(ctx, ctx.Command.Name, 1) // last argument is exit code
 	}
 
-	if ctx.IsSet("hard") && ctx.IsSet("fifo") {
-		fatalIf(errInvalidArgument(), "Only one of --hard or --fifo flags can be set")
-	}
-	if (ctx.IsSet("hard") || ctx.IsSet("fifo")) && len(ctx.Args()) == 0 {
+	if (ctx.IsSet("hard")) && len(ctx.Args()) == 0 {
```

Comment:
len(ctx.Args()) == 0 is not needed, we already exit in the beginning of the function if no args is passed